### DISCUSSION TITLE
The media could not be loaded, either because the server or network failed or because the format is not supported.

### DISCUSSION THREAD

**Discussion by thereal-BLKMGK** [mrlt8/docker-wyze-bridge#1198] [Q&A]
I've started getting this in the WEB UI, version before last I believe. I'm currently up to date on v2.9.1. I'm sure there's some setting I've screwed up to get stuck here and it doesn't appear on all of my feeds. This happens both on local and on remote feeds (remote feeds VERY important to me BTW). I'm running this on unRAID in a container, API keys enabled and WEB Auth turned off. Thoughts? I feed BlueIRIS with this but do pay Wyze for their service as well.

**Comment by nforceroh** [comment]
Getting the same issue on 2.9.1 also but one of my wyze cam v3 works but none of the other v3 and v2's work. all say format is unsupported. afaik, all cameras are on latest firmware

**Reply by nforceroh** [reply]
I just checked my firmware, My working v3 is on 4.36.3.18, and my other v3 not working is at 4.36.11.8391 and my v2 are at 4.9.9.3006
As a test I sacrificed my working v3 and updated the firmware and now it is reporting the format is not supported

**Reply by mrlt8** [reply]
Anything in the logs? Can you try setting `ON_DEMAND=False` to see if that helps keep the connection alive? 

There is currently an issue connecting to remote cameras on 2024 firmware, though it seems to work if you can let the bridge access the camera's local IP via a VPN connection.

**Comment by thereal-BLKMGK** [comment]
Remote cams failed when firmware was updated to latest firmware months ago. I traveled to the remote site and downgraded all of my cameras and things were good - for awhile. 4.36.11.7071 runs on those V3 cameras with a V3 PTZ camera running 4.50.4.7252. I have a Flood Pro running 2.0.921 also that's not been added to BlueIris yet.

Locally I have a V3 running 4.61.0.3 (RTSP but turned off), 2x V3 running 4.36.11.8391, a Cam Pan running 4.10.9.2847, and a V2 running 4.28.4.41 (RTSP turned on). ALL of the cameras are having issues on the Wyze-Bridge status page :(

In the logs I see audio pipe closing messages in yellow and "[cam-name] [CONTROL] ERROR - error='[-20029] AV_ER_CLEANBUF_ALREADY_CALLED', cmd='_bitrate'" messages showing up in red. My flood cam is throwing an odd error occasionally too -> [floodlight-cam] [-10] IOTC_ER_UNLICENSE and then WyzeBridge says it will cool down that cam.

Happy to try tweaks and help with troubleshooting although my remote cams cannot be easily accessed. Flashing firmware and making changes locally is easy however and I could also take one of my cams elsewhere to troubleshoot remote access issues too. I have a V4 and a new PTZ I've yet to setup. I note that all of the snapshots displayed on the status page show 5/13 or 5/14 but I'm not sure that means much as this has been an issue longer than that. If the dev would like access to one or more of the cams I can likely help with that. I get alerts and things just fine from the cams and can use my Wyze service fine on all of them but my BI system has proven valuable to catch things the service misses....

**Reply by thereal-BLKMGK** [reply]
Updated to v2.9.2 tonight. All of the preview screens on the web page went blank and no video available but BlueIris got good video and looking at the logs it was looking great! I was able to pull up a separate HLS stream which opened in another tab. I clicked on Snapshot which brought up snapshots of the last alerts, then clicked video again and... it looks up! Not been very long but so far so good, this is looking great! When a stream goes down now itlooks to be saying "Playback cannot continue. No available working or supported playlists" so when it's going down it looks to be actual connectivity issues - much better I think :)

**Reply by thereal-BLKMGK** [reply]
This morning streams for most but not all cameras were down in BlueIris. I refreshed the Wyze-Bridge page and many of the streams came back up in BI but only some in the web page - all of them local but NOT all of the local cameras. I'm wondering if leaving the status page up all the time is a good idea or if closing it would change anything? What do other's do? Is anyone else seeing better results with this version? IMO it's an improvement for sure!

**Reply by mrlt8** [reply]
Could you see if setting ON_DEMAND to false helps?

**Reply by thereal-BLKMGK** [reply]
ON_DEMAND=False is already set. Since I attempt to do constant recording with BlueIris I had set this as soon as I spotted it. I keep a log window open/running and would be happy to share whatever might be helpful for sure! Currently all cams down in BI but a quick recycle of the container will bring them back for a good while usually. I might be willing to share streams or cams too if that might help although obv that would only be  remote stream troubleshooting - no easy way to get onto my network heh. Both local and Remote cams seem to have issues after a time. I can likely get the Docker settings for you too, unRAID makes this a little harder but not impossible :)

**Reply by mrlt8** [reply]
hmm, can you post the relevant logs around the any of the `stream is down` messages?

I believe some users reported having some connection issues when running in unRaid. Any chance you can try running the container using regular docker/docker-compose?

**Reply by thereal-BLKMGK** [reply]
I'll look for that inthe logs. Currently BI is showing multiple streams down (but not all!), the Wyze-Bridge page shows all but one or two with "The media could not be loaded, either because the server or network failed or because the format is not supported." (including on one stream that BI is showing - weird), and my logs show a great deal of thjis -> https://pastebin.com/hR9cE6Dv

Kitchen counter is the only one of the cameras in that list that's showing a stream. There's 2 others that didn't show in that short list that are also streaming. I did see Kitchen Counter throw a "super slow" entry too. It doesn't ever indicate the stream is down. Refreshed the bridge web page and  cams showing in BI came up, the rest still show media unable to load.

I can likely run this setup on another machine not on unRAID but it may take me a day or two. unRAID has provided awesome training wheels for containers and I'll have to work it out on a Kali box I've got lol. I'll try to do that ASAP and post results. Hopefully this isn't an unRAID issue - ugh! Appreciate the support, stay tuned!

**Reply by thereal-BLKMGK** [reply]
Not yet tried on new hardware - will try this weekend. However, did update to the newest version (2.93) :) A note about the vid windows showing messages about unsupported format - tried in Edge and it worked for those windows. On a hunch I tried in a NEW Chrome tab - working! Currently after the update ALL streams are recording in BI, some are buffering in the web page but connectivity could explain that easily - remote cams . Just wanted to mention where I'm at, hopefully others are testing on the new code too!

**Reply by thereal-BLKMGK** [reply]
Running it on another box now, even with the new version I was having the video drop from Blue Iris. This setup isn't quite apple to apples though as audio isn't enabled. The install of Docker I've got isn't taking in the YAML file so it's all being setup from the commandline and I'm fairly newb with Docker at the commandline. So far so good though. I'm NOT seeing issues in the browser windows about video formats, a setting maybe? I'm not sure. I don't know how to access the logs and it's not daemonized. I'll watch and see how it goes, try to report back in a few days!

**Reply by wookieboy** [reply]
I am getting this same `IOTC_ER_UNLICENSE` error message on all of my cameras. 100% new here and was trying to get these cameras to integrate into my network using something other than the wyse app. 

I have a few different kinds of cameras and firmwares installed.
```
OG - 1.0.80
Doorbell Pro - 1.0.73
Floodlight Pro - 2.2.1.1135
Battery Cam Pro - 1.3.16.1
Battery Cam Pro - 1.4.1.2288
```

They all give the same message over and over as long as I try to stay connected. I've tried RSTP / HLS / RTMP and all of them give the same result. I was first typing this docker bridge inside of homeassistnat and I was not getting any helpful logs so I loaded it up with docker-compose and watched the stdout. 

```wyze-bridge  | [WyzeBridge] 🎉 Connecting to WyzeCam Floodlight Pro - Driveway on
wyze-bridge  | [driveway] [-10] IOTC_ER_UNLICENSE
wyze-bridge  | [WyzeBridge] Driveway will cooldown for 10s.
```
This message is on repeat doesn't matter which camera I am trying to access. Happy to test / provide logs / whatever I can do to assist the community.

**Reply by mrlt8** [reply]
@wookieboy none of those cameras are supported. Please see the [readme](https://github.com/mrlt8/docker-wyze-bridge?tab=readme-ov-file#supported-cameras) for a list of supported cameras.

**Reply by thereal-BLKMGK** [reply]
Been a few days, my supported firmware\cameras are working much better. The settings aren't quite the same but I'm no longer getting issues with unsupported media and if a camera does drop a refresh of the web page brings it back. The cameras reconnect way better on their own it seems so BI is keeping good track of the video. Audio being turned off seems to be the biggest change between the two installs (and the host not being unRAID). I'll try to bring the two installs closer to parity to see if that changes anything as the host should't make a difference - but could. Anyway, didn't want to leave folks hanging - with a separate host and simple install I'm getting pretty darn good results even on remote connected cameras (I've got older firmware to support that - I backflashed).
